File: Practice/13/python/13project.py
import math
n=2
i=int(input("Введите значение "))
while n<=math.sqrt(i):
    if i % n == 0: 
        print("Число ",i ," составное")
        break;
    else:
        print("Число", i, " простое")
        break;
  

File: Practice/13/c++/13project/13project/13project.cpp

#include <iostream>
#include <math.h>
using namespace std;
int main()
{
    setlocale(LC_ALL, "Russian");
    int i,n;
    cout << ("Введите целое положительное число") << endl;
    cin >> i;
    for (n=2;n<=sqrt(i);n++){
        if (i % n == 0) {
            cout << ("Число ") << i << (" составное") << endl;
            break;
            return(0);
        }
        else cout << ("Число ") << i << (" простое") << endl;
        break;
   }
}  

